lib/bundles/inspec-compliance/api.rb

```diff
@@ -36,6 +36,7 @@ def self.profiles(config)
           end
         end.flatten
       else
+        puts '401 Unauthorized. Please check your token.'
```

Comment:
good point, thanks!! ya, i'll rework this a bit later -- a case statement on the http response code sounds like the right thing to do there